src/classes/CRLP_Rollup_SEL.cls

```diff
@@ -236,11 +524,11 @@ public class CRLP_Rollup_SEL {
      * @return String list of fields to append to a SOQL query
      */
     public static String getReferencedFieldsByObjectAsString(SObjectType objType) {
-        String soql = '';
-        for (DescribeFieldResult dfr : getReferencedFieldsByObject(objType)) {
```

Comment:
- Changing non-descriptive variable name to something more descriptive.
- This method generates Fields for a SOQL SELECT statement.  Instead of continually adding on to a string and having to remove the final separator before returning, collects Fields in a `List<String>` and uses `String.join` at the end with the separator (which does not add the separator at the end.

Reply:
Very nice refactor